docs/index.md

```diff
@@ -1,17 +1,14 @@
-# Welcome to MkDocs
+# Welcome message
+Welcome, fellow architects, to the fascinating world of 'Smash Grade'! 
 
-For full documentation visit [mkdocs.org](https://www.mkdocs.org) and 
+Imagine a Swiss timepiece meticulously designed with intricate precision, our application is no different — a harmonious blend of technology and innovation. 
```

Comment:
Are you sure prose about watchmaking sends the right welcome message for developers checking out technical documentation of a grading app?
It probably has more effect as an epigram in the user manual.

Reply:
Ja stimmt, das können wir dann noch auslöschen 👍🏻